app/views/shared/_header.html.erb

```diff
@@ -3,8 +3,8 @@
     <%= link_to "PDC Describe", root_path, class: 'navbar-logo' %>
     <div class="menu">
       <nav id="menu" class="col-sm" role="menu">
-      <%= link_to "About", "/about", :class => "btn btn-primary", :role => "menuitem" %>
-      <%= link_to "How to Submit", "/how-to-submit", :class => "btn btn-primary", :role => "menuitem" %>
+      <%= link_to "About","#{root_url}about", :class => "btn btn-primary", :role => "menuitem" %>
+      <%= link_to "How to Submit", "#{root_url}how-to-submit", :class => "btn btn-primary", :role => "menuitem" %>
```

Comment:
I think the idiomatic way for this is:

```suggestion
<%= link_to "About",welcome_about_path, :class => "btn btn-primary", :role => "menuitem" %>
<%= link_to "How to Submit", welcome_how_to_submit_path, :class => "btn btn-primary", :role => "menuitem" %>
```
which uses the alias given to these pages (e.g. `welcome_about_path`) in `routes.rb`